blog/releases/announcing-knative-v0-22-release.md

```diff
@@ -0,0 +1,195 @@
+  - Check out the docs for more details at https://knative.dev/docs/eventing/broker/kafka-broker . [#589](https://github.com/knative-sandbox/eventing-kafka-broker/pull/)
+
+#### 🐞 Bug Fixes
+
+- Add producer interceptor io.cloudevents.kafka.PartitionKeyExtensionInterceptor to provide ordered delivery based on the partitioning extension of the CloudEvents spec. [#751](https://github.com/knative-sandbox/eventing-kafka-broker/pull/751)
+- Fix unable to deploy KafkaSink without Kafka Broker installed [#714](https://github.com/knative-sandbox/eventing-kafka-broker/pull/714)
+
+### Client v0.22
+
+kn 0.22.0 comes with some bug fixes and minor feature enhancements. It's mostly a polishing release. If you are using the client API, there is a breaking change that was needed to align with Kubernetes client API's
+
+#### META
+The compile dependencies have been updated to Knative Serving 0.22.0 and Knative Eventing 0.22.0. There are no changes in the used API version for communicating with the backend.
+
+#### Managing custom domain mappings
+
+This release adds support for CRUD management of domain mappings. I.e. you can use `kn domain` along with its sub-commands `create`, `update`, `describe`, `list` and `delete` to fully manage DomainMapping resources for the installation of custom domains for Knative services.
+
+```
+# Create a domain mappings 'hello.example.com' for Knative service 'hello'
+kn domain create hello.example.com --ref hello
+
+# Update a domain mappings 'hello.example.com' for Knative service 'hello'
+kn domain create hello.example.com --refFlags hello
+
+# List all domain mappings
+kn domain list
+
+# Delete domain mappings 'hello.example.com'
+kn domain delete hello.example.com
+```
+See `kn domain help` for more information.
+
+
+#### Client API signature change
+
+A `context.Context` object is added as the first argument to every Client API method to align with Kubernetes' client API signatures. This change does not affect any client's CLI usage, only if the client-specific Golang API is used for communicating with the Knative backend has changed. The migration is straight forward, either pass an already existing context down to the call or leverage one of the available standard contexts (like `context.TODO()`).
+
+This change does _not_ affect any CLI usage of the client.
+
+### CLI Plugins
+
+#### 💫 New Features & Changes
+
+The plugins that are released aligned with 0.22 are:
+
+- [kn-plugin-admin](https://github.com/knative-sandbox/kn-plugin-admin) for managing Knative installations that are running on Kubernetes | [download](https://github.com/knative-sandbox/kn-plugin-admin/releases/tag/v0.22.0)
+- [kn-plugin-source-kafka](https://github.com/knative-sandbox/kn-plugin-source-kafka) for managing a Kafka Source that has been installed via [eventing-kafka](https://github.com/knative-sandbox/eventing-kafka) on the backend | [download](https://github.com/knative-sandbox/kn-plugin-source-kafka/releases/tag/v0.22.0)
```

Comment:
The new plugins are mentioned more generally in a section earlier in the doc; let's group this information maybe? It seems confusing to have it in different places.

Reply:
They are mentioned in the first section `Highlights` in one sentence.
```
- There are two new CLI plugins align with 0.22 release, [kn-plugin-admin](https://github.com/knative-sandbox/kn-plugin-admin) and [kn-plugin-source-kafka](https://github.com/knative-sandbox/kn-plugin-source-kafka)
```

This section is the area with the content details about the plugins. 

Let me know if you want me to remove the sentence from the first section `Highlights`